privacy/company-subprocessors/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Company Subprocessors
+description: Basecamp LLC uses some other third-party subprocessors for company purposes outside of delivering our services.
+---
+
+# Company subprocessors
+
+Beyond the subprocessors used to deliver each of our products, we as a company also use other third-party software that may process your information under certain circumstances.
```

Comment:
Are we acting as a controller in these cases? May want to classify these as processors rather than subprocessors if so.

Reply:
Very good point - thank you for catching that. Corrected in https://github.com/basecamp/policies/pull/45/commits/1023ee9b206147e25a3494c177ae6ab8ab7c6346